tests/baselines/reference/api/tsserverlibrary.d.ts

```diff
@@ -8347,7 +8347,7 @@ declare namespace ts.server {
         excludedFiles: ReadonlyArray<NormalizedPath>;
         private typeAcquisition;
         updateGraph(): boolean;
-        getExcludedFiles(): readonly NormalizedPath[];
```

Comment:
@weswigham I didn't expect to see this show up here.

Reply:
Ahhhh, that's from the swap in gulp back to using the LKG for everything. The "remove jake" PR has the fix (accept the baseline, do a non-lkg validation build) already.